lib/polarshift/request.rb

```diff
@@ -176,14 +176,19 @@ def create_run(project_id:,
       end
 
       def create_run_smart(timeout: 360, **opts)
-        res = create_run(**opts)
-        if res[:exitstatus] == 202
-          op_url = JSON.load(res[:response])["operation_result_url"]
-          logger.info "to check operation status manually, you can: " \
-            "curl '#{op_url}' -u user:thepassword"
-          pr = wait_op(url: op_url, timeout: timeout)
-        else
-          raise %Q{got status "#{res[:exitstatus]}" creating a new test run:\n#{res[:response]}}
+        for retries in 1..5 do
+          res = create_run(**opts)
+          if res[:exitstatus] == 202
```

Comment:
Thank you @liangxia , can you elaborate what `202` represents here and why we raise an exception if it's not `202`?

Reply:
```
[09:27:00] INFO> HTTP POST user@https://POLARSHIFT-URL/project/OSE/run
[09:27:02] INFO> HTTP POST took 1.086 sec: 202 Accepted | application/json 267 bytes
```
This is the `202` comes from.

Reply:
OK, that seems to be the original code here.

Reply:
It's explained in https://gitlab.cee.redhat.com/aosqe/polarshift/-/blob/master/doc/API.adoc#create-test-run